plugins/source/typeform/plugin/plugin.py

```diff
@@ -0,0 +1,63 @@
+import json
+from typing import List, Generator
+
+from cloudquery.sdk import message
+from cloudquery.sdk import plugin
+from cloudquery.sdk import schema
+from cloudquery.sdk.scheduler import Scheduler, TableResolver
+
+from plugin import tables
+from plugin.client import Client, Spec
+
+PLUGIN_NAME = "typeform"
+PLUGIN_VERSION = "0.0.1"  # {x-release-please-version}
+
+
+class TypeformPlugin(plugin.Plugin):
+    def __init__(self) -> None:
+        super().__init__(PLUGIN_NAME, PLUGIN_VERSION)
+        self._spec_json = None
+        self._spec = None
+        self._scheduler = None
+        self._client = None
+
+    def init(self, spec_bytes):
+        self._spec_json = json.loads(spec_bytes)
+        self._spec = Spec(**self._spec_json)
+        self._spec.validate()
+        self._scheduler = Scheduler(self._spec.concurrency, self._spec.queue_size)
+        self._client = Client(self._spec)
+
+    def get_tables(self, options: plugin.TableOptions) -> List[plugin.Table]:
+        all_tables: List[plugin.Table] = [
+            tables.Forms(),
+        ]
+
+        # set parent table relationships
+        for table in all_tables:
+            for relation in table.relations:
+                relation.parent = table
```

Comment:
Was this needed or just 'nice to have'? I don't remember doing this in the Square plugin, which seemed to work (at least query) with child tables.

Reply:
It's necessary for the docs to indicate the parent-child relationship on the child table